<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CircleWidget</class>
 <widget class="QWidget" name="CircleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxDirect">
   <property name="geometry">
    <rect>
     <x>667</x>
     <y>510</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>方向</string>
   </property>
   <widget class="QRadioButton" name="radioButtonW2B">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>亮到暗</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonB2W">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>暗到亮</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>641</width>
     <height>161</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>坐标调整</string>
   </property>
   <widget class="QPushButton" name="btnAddRadius">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>94</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubRadius">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>94</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCenterX">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCircleDown">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>114</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCenterY">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCircleLeft">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>中心点 X:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCircleRight">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCircleUP">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>26</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>上</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>中心点 Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRadius">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>45</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>40</y>
      <width>36</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>半径:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
